Source/UI/NewUI/PaymentForm/CardScheme.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CardScheme.swift
+//  
+//
+//  Created by Ehab Alsharkawy
+//  Copyright © 2022 Checkout. All rights reserved.
+
+import Checkout
+
+public enum CardScheme {
+  case unknown
+  case mada
+  case visa
+  case mastercard
+  case maestro
+  case americanExpress
+  case discover
+  case dinersClub
+  case jcb
+
+  var checkoutScheme: Card.Scheme {
+    switch self {
+      case .unknown: return .unknown
+      case .mada: return .mada
+      case .visa: return .visa
+      case .mastercard: return .mastercard
+      case .maestro: return .maestro
+      case .americanExpress: return .americanExpress
+      case .discover: return .discover
+      case .dinersClub: return .dinersClub
+      case .jcb: return .jcb
+    }
+  }
+}
```

Comment:
```suggestion
public typealias Card = Checkout.Card
```

Reply:
Nice observation, I hadn't known this works 👏 . 

Just verified this trying in my current application where I have 3 linked projects and doing this at the middle layer did result in top object getting all functionality without actually importing the bottom layer, just via the interface.

Definitely looks useful if there is no added value and its a 100% match just passing through 👍